docs/concepts/executor/serve.md

```diff
@@ -59,7 +58,8 @@ class MyExec(Executor):
 
 
 with Deployment(uses=MyExec, port=12345, replicas=2) as dep:
-    dep.block()
+    docs = dep.post(inputs=DocumentArray.empty(1), on='/foo')
+    print(docs.texts)
 ```
```

Comment:
example of output?

Reply:
<img width="812" alt="image" src="https://user-images.githubusercontent.com/15269265/211591132-50e7fe35-e62e-4351-a43e-5727f3e13dce.png">
already there for each type
https://feat-add-client-deployment--jina-docs.netlify.app/concepts/executor/serve/#serve-directly